enterprise/server/util/dynamic/dynamic.go

```diff
@@ -0,0 +1,80 @@
+package dynamic
```

Comment:
lol ok ok naming is very hard so no shade here but I feel like this is a test :P

dynamic feels... very generic and it's not at all clear to me what this thing does or why someone might want to use it. What if you called it "fieldgetter" or something more descriptive instead?